amm/drink-tests/src/stable_swap_tests/tests_add_remove_lp.rs

```diff
@@ -0,0 +1,400 @@
+use drink::{self, session::Session};
+
+use super::*;
+
+// ref https://github.com/ref-finance/ref-contracts/blob/d241d7aeaa6250937b160d56e5c4b5b48d9d97f7/ref-exchange/tests/test_stable_pool.rs#L123
+#[drink::test]
+fn test_01(mut session: Session) {
+    seed_account(&mut session, CHARLIE);
+    seed_account(&mut session, DAVE);
+    seed_account(&mut session, EVA);
+    let initial_reserves = vec![100000 * ONE_DAI, 100000 * ONE_USDT, 100000 * ONE_USDC];
+    let (stable_swap, tokens) = setup_stable_swap_with_tokens(
+        &mut session,
+        vec![18, 6, 6],
+        initial_reserves
+            .iter()
+            .map(|amount| amount * 100_000_000_000)
+            .collect(),
+        10_000,
+        25,
+        2000,
+        BOB,
+    );
+
+    _ = stable_swap::add_liquidity(
```

Comment:
In `setup_stable_swap_with_tokens` it's hidden that it's `BOB` that gets all the "moni" and it increases the allownce on `stable_swap`.